scripts/autotools/libmongoc/CheckResolv.m4

```diff
@@ -80,7 +84,9 @@ AC_LINK_IFELSE([AC_LANG_PROGRAM([[
    ]])], [
       AC_MSG_RESULT([yes])
       AC_SUBST(MONGOC_HAVE_RES_SEARCH, 1)
-      found_resolv="yes"
```

Comment:
This might create an inconsistency where `found_resolv` should be `yes`, but we're not setting it. However, I believe we can completely remove the variable since we're not checking for it anywhere else.